pages/clusters/overview.md

```diff
@@ -30,8 +52,38 @@ _Clusters_ will now appear in the global navigation.
 
 Enabling clusters will not impact any of your existing agents or pipelines, nor will you require any workflow-breaking changes for you to try clusters.
 
-Once you’ve enabled clusters, all members of your organization will see _Clusters_ in the global navigation. This will show all your clusters as well as _Unclustered_ agents and pipelines.
+Once clusters is enabled, when selecting _Agents_ in the global navigation your will be shown all your clusters as well as _Unclustered_ agents and pipelines.
 
 Any agents or pipelines not associated with a cluster are called _unclustered_. To view and manage your unclustered agents, agent tokens, and pipelines, select _Unclustered_.
 
-To view all running agents in your organization (in a cluster or not), click on _All agents_ in the sidebar.
+## Clusters and Queues best practice
```

Comment:
I know we were talking about this yesterday, but what do you think about the idea of moving some of this content into the new 'Managing queues in a cluster' (to be created as part of https://github.com/buildkite/docs/pull/2642) on the 'Managing clusters' page of the docs?

Reply:
As discussed with Liam, will/may rework this into #2642